index.html

```diff
@@ -68,16 +68,16 @@ <h1 class="splash">
                         <div class="col-xs-12">
                             <h2>About nodeday 2015</h2>
```

Comment:
I know this isn't technically part of your PR, but would you mind changing it too?
We've had several events on 2015, and we should start segmenting by geography.